{"uuid": "8070e12c-23b9-40cf-b5a3-5a41d08724b9", "children": ["21e9c15e-9e19-408b-81a6-2dcf044bd00b"], "befores": [{"name": "_xunit_setup_method_fixture_TestSetup2", "status": "broken", "statusDetails": {"message": "KeyError: 1\n", "trace": "  File \"/Users/aymanelsayeed/.local/share/virtualenvs/AllureExample-wL7UUQNi/lib/python3.9/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/aymanelsayeed/.local/share/virtualenvs/AllureExample-wL7UUQNi/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/aymanelsayeed/.local/share/virtualenvs/AllureExample-wL7UUQNi/lib/python3.9/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/aymanelsayeed/.local/share/virtualenvs/AllureExample-wL7UUQNi/lib/python3.9/site-packages/_pytest/python.py\", line 899, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"/Users/aymanelsayeed/.local/share/virtualenvs/AllureExample-wL7UUQNi/lib/python3.9/site-packages/_pytest/python.py\", line 777, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/aymanelsayeed/IdeaProjects/AllureExample/tests/tests_setup/test_steup_2.py\", line 12, in setup_method\n    Collection.pop(self.counter)\n"}, "start": 1722076912431, "stop": 1722076912431}], "start": 1722076912431, "stop": 1722076912445}